gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=1.1.28_2
+version=1.1.28.2
```

Comment:
why are we changing the version from `1.1.28_2` to `1.1.28.2`?

Reply:
To be consistent with 1.1.28.1
https://github.com/salesforce/carbonj/blob/3226d4cee0ce1c6f9c8f652f547bce8551159fc0/gradle.properties#L1